sdk/go/auto/optpreview/optpreview.go

```diff
@@ -101,6 +101,13 @@ func UserAgent(agent string) Option {
 	})
 }
 
+// Plan specifies the path to save the update plan from the preview to.
```

Comment:
```suggestion
// Plan specifies the path where the update plan should be saved.
```
Should we call this `PlanPath` instead of `Plan`?